test/fixtures/mediawiki/invalid.js

```diff
@@ -3,4 +3,7 @@
 
 	// eslint-disable-next-line mediawiki/msg-doc
 	mw.msg( 'foo-' + bar );
+
+	// eslint-disable-next-line mediawiki/valid-package-file-require
+	require( 'foo' );
```

Comment:
this is valid - if you aren't getting an unused directive error that is a problem...

Reply:
Turns out the plugin includes a file-exists check, so we need to create "foo.js" then include it incorrectly to make it fail.